examples/always_returning_example.py

```diff
@@ -0,0 +1,21 @@
+def loop():
+    for j in range(0, 1):
```

Comment:
Emmm, do you think that may be a little bit confusing for students? If they see two messages indicating the same error. 
Currently, the checker is reporting specifically the location of the error(the closest loop).
What do you think?